.github/workflows/ci.yml

```diff
@@ -81,74 +89,94 @@ jobs:
             clang-runtime: '16'
             cling: Off
             cppyy: Off
-          - name: win2022-msvc-cling
+          - name: win2022-msvc-cling-18
+            os: windows-2022
+            compiler: msvc
+            clang-runtime: '18'
+            cling: On
+            cling-version: 'refs/heads/master'
+          - name: win2022-msvc-cling-16
             os: windows-2022
             compiler: msvc
-            clang-runtime: '13'
+            clang-runtime: '16'
```

Comment:
Please remove the clang llvm 16 jobs as this will overwhelm the llvm cache. We need to just keep the latest version of llvm with respect to cling.

Reply:
We need both. Cling differs with the LLVM-18 upgrade and ROOT requires this check to be made to support releases >6.32 . I would rather drop clang-repl16

Reply:
@aaronj0 We need to drop one job. If you think it would be better to drop clang-repl-16 then I am ok with them. We can continue to support it in the other repos for now since they don't have any Windows builds filling in the cache.